### PR TITLE
test: add test for disabled code lens branch

### PR DESCRIPTION
**Pull request by is0692vs**
Added a test in `src/test/inlineCommands.unit.test.ts` to verify that `JulesCodeLensProvider.provideCodeLenses` skips symbol lookup and returns no lenses when disabled via configuration. This addresses code coverage for the disabled configuration edge-case.

---
*PR created automatically by Jules for task [2334577690087889489](https://jules.google.com/task/2334577690087889489) started by @is0692vs*

<details open><summary><h3>Greptile Summary</h3></summary>

無効化設定時に `JulesCodeLensProvider.provideCodeLenses` がシンボル検索をスキップして空配列を返すブランチをカバーするテストを1件追加しています。テストのロジック（`getConfiguration` スタブ・`executeCommand` の未呼び出し検証・`dispose` 呼び出し）は本番コードの早期リターンパスを正確に反映しており、セミコロンやトレーリングカンマなどのスタイルもファイル全体と一貫しています。
</details>

<details open><summary><h3>Confidence Score: 5/5</h3></summary>

マージ安全。テスト専用の変更であり、本番コードへの影響はありません。

P0・P1の問題は見当たりません。テストロジックは本番コードの早期リターンパスを正確に検証しており、スタブ設定・アサーション・クリーンアップのいずれも適切です。

特に注意が必要なファイルはありません。
</details>

<details open><summary><h3>Important Files Changed</h3></summary>

| Filename | Overview |
|----------|----------|
| src/test/inlineCommands.unit.test.ts | 無効化時のコードレンズブランチをカバーする新テストを追加。ロジックは正確で、コーディングスタイルもファイル全体と一貫している。 |

</details>

</details>

<h3>Sequence Diagram</h3>

```mermaid
sequenceDiagram
    participant Test
    participant Provider as JulesCodeLensProvider
    participant Config as vscode.workspace.getConfiguration (stub)
    participant Cmd as vscode.commands.executeCommand (stub)

    Test->>Config: stub → get() returns false
    Test->>Provider: new JulesCodeLensProvider(logChannel)
    Test->>Provider: provideCodeLenses(document, token)
    Provider->>Config: getConfiguration("jules-extension").get("enableCodeLens", true)
    Config-->>Provider: false
    Provider-->>Test: return [] (early exit, executeCommand skipped)
    Test->>Test: assert lenses === []
    Test->>Cmd: assert executeStub.called === false
    Test->>Provider: dispose()
```

<sub>Reviews (3): Last reviewed commit: ["style: add semicolons to JulesCodeLensPr..."](https://github.com/hiroki-org/jules-extension/commit/ec9c0904b2a75c8c42c9b6b549aa3e228de8e0f7) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=30547909)</sub>

### CODE REVIEW

File: src/test/inlineCommands.unit.test.ts [hunk 1]
```diff
@@ -70,6 +70,23 @@ suite("inlineCommands Test Suite", () => {
...
+        const provider = new JulesCodeLensProvider({ appendLine: sandbox.stub() } as any)
+        const lenses = await provider.provideCodeLenses(
+            { uri: vscode.Uri.parse("file:///sample.ts") } as any,
+            { isCancellationRequested: false } as any,
+        )
+
+        assert.deepStrictEqual(lenses, [])
+        assert.strictEqual(executeStub.called, false)
+        provider.dispose()
+    })
```
[the remark this answers is not in the document]
Comment (is0692vs) [reply src/test/inlineCommands.unit.test.ts]: 対応しました: 指摘事項を修正しました。検証: ユニットテスト通過済み。

[the remark this answers is not in the document]
Comment (is0692vs) [reply src/test/inlineCommands.unit.test.ts]: 対応しました: 指摘事項を修正しました。検証: ユニットテスト通過済み。

**Comment by is0692vs** [comment]
レビュー指摘対応と thread resolve を完了しました。CI/コンフリクト確認済みです。再レビューお願いします。

**Comment by is0692vs** [comment]
対応しました: セミコロルを全ステートメントの末尾に追加しました。変更内容: test block内の14行目から27行目のステートメントすべてにセミコロルを追加（closing `});`も含む）。

影響範囲: src/test/inlineCommands.unit.test.ts のtest block のみ。動作は変わらず、style統一のみ。

検証: pnpm run test:unit で563テスト全て成功、pnpm run lint でESLint エラーなし。

コミット: 0b4c1d3

**Comment by is0692vs** [comment]
レビュー指摘対応と thread resolve を完了しました。CI/コンフリクト確認済みです。再レビューお願いします。

**Comment by is0692vs** [comment]
@greptile

**Comment by is0692vs** [comment]
コンフリクト解決と rebase 完了しました。CI 再実行中です。再レビューお願いします。

**Comment by is0692vs** [comment]
@greptile
